src/test_workflow/benchmark_test/benchmark_test_cluster.py

```diff
@@ -143,12 +141,12 @@ def setup_cdk_params(self, config: dict) -> dict:
             artifact_url = self.manifest.build.location if isinstance(self.manifest, BundleManifest) else \
                 f"https://artifacts.opensearch.org/snapshots/core/opensearch/{self.manifest.build.version}/opensearch-min-" \
                 f"{self.manifest.build.version}-linux-{self.manifest.build.architecture}-latest.tar.gz"
-            if not self.args.insecure and semver.compare(self.manifest.build.version, '2.12.0') != -1:
-                need_strong_password = True
+            if not self.args.insecure:
+                password = 'myStrongPassword123!' if get_password(self.manifest.build.version) == 'myStrongPassword123!' else None
         else:
             artifact_url = self.args.distribution_url.strip()
-            if not self.args.insecure and semver.compare(self.args.distribution_version, '2.12.0') != -1:
-                need_strong_password = True
+            if not self.args.insecure:
+                password = 'myStrongPassword123!' if get_password(self.args.distribution_version) == 'myStrongPassword123!' else None
 
```

Comment:
Same as abobe.

Reply:
I was also confused about this, but it seems like this specific test is trying to set "myStrongPassword123!" or None depending on the version, instead of "myStrongPassword123!" or "admin". Thus, I had to make the usage of the util function in this way.

Reply:
The comparison with an actual password is strange. Why is this hard-coded and not configuration (which can be anything, including nothing)?

Reply:
Agree, just move all the processing to util and return final password.